services/ec2/src/main/resources/emails/accessGranted.ftl

```diff
@@ -33,11 +33,15 @@
             <blockquote>No reason was provided</blockquote>
         </#if>
     </div>
-
+    <div>
+        <p style="color: darkred">
+            ${changeDisclaimer}
```

Comment:
Can we update the template to not generate this block of code if the changeDisclaimer is not set?